pages/analysis.md

```diff
@@ -1,15 +1,132 @@
+trade. As per the documented information, the slave trade was extremely expansive
+that the incursion by privateers had little to no effect on the market itself. 
+
+The images do raise questions to the effects privateers and pirates had on smaller
+countries. These smaller states would not have had such availability to the slave 
+trade if not provided by the captive ships. The images below demonstrate the 
+disparities and theory.
+
+---
+
+![image]({{ site.baseurl}}/assets/images/1.%20Origin%20Ports.png)
+	
+This image represents most of the documented ports where slave 
+ships originated. To the left (black dots) the image is of all the ships
+that departed and were never held captive and taken over by privateers or 
+pirates. The image to the right (purple dots) shows the ports where ships
+launched and eventually were taken over by privateers or pirates. The
+purpose of these images is to show that at this point there is no significance
+between pirated ships and ships that completed their voyage without being 
+impeded. The theory is that to this point pirates and privateers were more 
+opportunistic scavengers of ships and less methodical in their selection
+
+---
+
+![image]({{ site.baseurl}}/assets/images/2.%20To%20slave%20ports.png)
+
+The path of travel represents paths as the crow flies, although 
+traveling ships make the paths. The quantity of activity is a representation 
+of over one hundred years but it is easy to see the differences between the 
+magnitude of business conducted by ships that were never affected by pirates 
+(left image, black lines) and the vessels that will eventually be affected 
+by privateers or pirates (right image, purple lines). Here it can be viewed 
+how little effect ships captured by these two parties could have had on the 
+slave trade. There is still no change in ships and their actions to this point. 
+On this image the ships left their home ports and headed to the ports where 
+they will be loading slaves onboard. 
+
+---
+
+![image]({{ site.baseurl}}/assets/images/3.%20To%20slave%20drop%20off%20point.png)
+
+The slaves are picked up in Africa and journeyed to the Caribbean 
+and West Indies region. Until this point, there has been little to no change
+in pattern comparing ships never captured to those that were eventually affected 
+by privateers or pirates. This image represents the path the vessels took to reach
+the slave market ports. The change in behavior occurs in this image. The left image 
+shows the mass amount of ships porting into the large slave market hubs. The right 
+image shows slaves being dropped off at similar areas but in more sporadic ports 
+compared to themasses. The interdiction of ships seems to have occurred closer to
+the Caribbean and West Indies region.  	
+	
+---
+
+![image]({{ site.baseurl}}/assets/images/4.%20Main%20Ports%20of%20Drop%20off.png)
+
+A clear visual of the ports where the slaves were dropped off is available in this
```

Comment:
Why not put the number of records for the label in these pictures, it would make it easier to grasp the volume we are discussing.